alchemy-web/docs/providers/supabase/bucket.md

```diff
@@ -0,0 +1,207 @@
+# Supabase Bucket Resource
+
+The `Bucket` resource manages Supabase Storage buckets, which provide file storage capabilities with access controls.
+
+## Usage
+
+```typescript
+import { Bucket } from "alchemy/supabase";
+
+// Create a new storage bucket
+const bucket = Bucket("user-uploads", {
+  projectRef: "proj-123",
```

Comment:
Replace projectRef with `project: string | Project`

Make this consistent across the entire api surface area